internal/output/writer.go

```diff
@@ -0,0 +1,96 @@
+package output
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/anchore/syft/syft/format"
+	"github.com/anchore/syft/syft/sbom"
+	"github.com/hashicorp/go-multierror"
+)
+
+type StreamWriter struct {
```

Comment:
`StreamWriter` appears to be an internal concern for this package since `MakeWriter` returns `sbom.Writer` so should be unexported

Reply:
It is.. except as I've structured the tests, I need to reference it in another package